signer/src/service/api/request.rs

```diff
@@ -0,0 +1,44 @@
+use mc_full_service::json_rpc::{
+    json_rpc_request::JsonRPCRequest, v2::models::tx_proposal::UnsignedTxProposal,
+};
+
+use mc_transaction_signer::types::TxoUnsynced;
+use serde::{Deserialize, Serialize};
+use std::convert::TryFrom;
+use strum::{EnumIter, IntoEnumIterator};
+
+pub fn help_str() -> String {
+    let mut help_str = "Please use json data to choose api commands. For example, \n\ncurl -s localhost:9090/wallet/v2 -d '{\"method\": \"create_account\", \"params\": {\"name\": \"Alice\"}}' -X POST -H 'Content-type: application/json'\n\nAvailable commands are:\n\n".to_owned();
+    for e in JsonCommandRequest::iter() {
+        help_str.push_str(&format!("{e:?}\n\n"));
+    }
+    help_str
+}
+
+impl TryFrom<&JsonRPCRequest> for JsonCommandRequest {
+    type Error = String;
+
+    fn try_from(src: &JsonRPCRequest) -> Result<JsonCommandRequest, String> {
+        let src_json: serde_json::Value = serde_json::json!(src);
```

Comment:
Why is this hack needed?

Reply:
Why do you consider this a hack? It's the same way we do it in Full Service

Reply:
Going from one Rust struct to another via JSON feels hacky, why can;t they be converted with standard From or TryFrom trait implementations?

Reply:
Oh I see what you mean, I think this may just be an old implementation from the early days of Full Service, I'll update that now :)

Reply:
Yeah, I see what you mean. I'll see if I can update this

Reply:
Discussed offline, while hacky it doesn't seem like there's an easier solution that doesn't involve a lot of typing...